pkg/driver/common/common.go

```diff
@@ -53,6 +55,7 @@ type DeviceStorage struct {
 	StorageKeys []byte
 	Serial      string
 	Onboard     *x509.Certificate
+	Options     []byte
```

Comment:
Looks like this is a json blob. Maybe that should be documented here?

Reply:
I put a comment to describe content